<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_staticClass</class>
 <widget class="QMainWindow" name="qt_staticClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_static</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>视频路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="read_file">
       <property name="text">
        <string>Read File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>81</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>FPS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_static.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>qt_staticClass</receiver>
   <slot>start_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>qt_staticClass</receiver>
   <slot>exit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>read_file</sender>
   <signal>clicked()</signal>
   <receiver>qt_staticClass</receiver>
   <slot>read_file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_clicked()</slot>
  <slot>exit_clicked()</slot>
  <slot>read_file_clicked()</slot>
 </slots>
</ui>
